### ISSUE TITLE
In High Contrast Mode, Keyboard focus is similar to Controls Boundary

### ISSUE THREAD

**Issue by crutkas** [microsoft/microsoft-ui-xaml#6267]
### Describe the bug

https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01
https://github.com/microsoft/PowerToys/issues/13469

Accessability team flagged PowerToys on a behavior that is stock HC behavior.  Settings has same behavior as well.  Talking with Chigusa, we decided it should be flagged here and so we can come to a resolution.

### Steps to reproduce the bug

1. Launch 'PowerToys' application.
2. Navigate to 'PowerRename' and activate it.
3. Turn on High Contrast mode.
4. Navigate to the Controls using Keyboard tab Navigation.
5. Observe the issue.

### ✔️ Expected Behavior

The Keyboard Focus should be highlighted with a different color. and it should be similar to boundary of the controls

### ❌ Actual Behavior

The Boundary of the controls are similar to Keyboard focus of the Controls. 

### Screenshots

https://user-images.githubusercontent.com/91203131/134922425-d674379a-2ea3-4160-a85b-176b56f94407.mp4

### NuGet package version

Microsoft.UI.Xaml 2.7.0

### Windows app type

- [X] UWP
- [X] Win32

### Device form factor

Desktop

**Comment by StephenLPeters** [comment]
@chigy FYI on the accessibility concern  @JJBrychell is there a way to modify the focus rect's color in these high contrast modes? what tools do we have to address this?

**Comment by crutkas** [comment]
Does this affect winui3?  We have since upgraded to that but I’m going to assume it does
